packages/components/src/components/Progress/ProgressTaskItem.tsx

```diff
@@ -0,0 +1,66 @@
+import { View } from "react-native";
+import React from "react";
+import { CircularProgress } from "../ProgressIndicator";
+import { Icon } from "../Icon";
+import { Typography } from "../Typography";
+import { ProgressStatus, ProgressTask } from "./types";
+import { EDSStyleSheet } from "../../styling";
+import { useStyles } from "../../hooks/useStyles";
+import { useToken } from "../../hooks/useToken";
+import { statusToColor, statusToIconName } from "./progressUtils";
+
+type ProgressTaskProps = {
+    task: ProgressTask;
+    status: ProgressStatus;
+};
+
+export const ProgressTaskItem = ({ task, status }: ProgressTaskProps) => {
+    const styles = useStyles(themeStyles);
+    const token = useToken();
+
+    const taskInProgress = status === "inProgress";
+
+    const renderErrorDetails = ({ errorDetails }: ProgressTask) => {
+        return errorDetails ? (
+            <View style={styles.errorDetailsContainer}>
+                <Typography>{errorDetails.message}</Typography>
```

Comment:
Is it intentional to render the error text as normal colored paragraph typography?
Visually it seems a little bit intense.

Reply:
I am not saying we should fix it if this is what the designs require. If they say it should be like this, it is completely fine to merge this PR and fix it afterwards imo